src/technologies/w.json

```diff
@@ -31,6 +31,18 @@
     "scriptSrc": "w3counter\\.com/tracker\\.js",
     "website": "https://www.w3counter.com"
   },
+  "Wake": {
+    "cats": [
+      6
+    ],
+    "description": "Wake is an ecommerce software with integrated digital solutions to manage online stores.",
+    "headers": {
+      "X-Powered-By": "Wake"
+    },
+    "icon": "Wake.png"
```

Comment:
```suggestion
    "icon": "Wake.png",
```

Reply:
@Angeesantos are you able to apply this fix?

Reply:
Hi Rick!

Just did it! Can you check please?